simple-facebook-instant-articles.php

```diff
@@ -542,7 +542,7 @@ public function render_images( \DOMDocument &$dom, \DOMXPath &$xpath ) {
 
 			// If image node is not a direct child of the body, we need to move it there.
 			// Recurse up the tree looking for highest level parent/grandparent node.
-			while ( $top_node->parentNode && 'body' !== $top_node->parentNode->nodeName ) {
+			while ( $top_node->parentNode && 'body' !== $top_node->parentNode->tagName ) {
```

Comment:
@mattheu I'm a bit unsure about this.. can't seem to see `tagName` property on `DOMNode` class?